### DISCUSSION TITLE
Running Aging cycles with Okane parameter set

### DISCUSSION THREAD

**Discussion by Dhawal-Goyal** [pybamm-team/PyBaMM#2780] [Q&A]
Hello I am running a aging cycles using Okane parameter set with SEI, lithium plating and  LAM options enabled as follows:

options = {"SEI": "ec reaction limited","lithium plating": "partially reversible", "loss of active material": "stress and reaction-driven"}
dfn = pybamm.lithium_ion.DFN(options = options)
params = pybamm.ParameterValues('OKane2022')

the experiments has been set up as below:

drive_cycle_current = pd.read_csv("US06.csv", comment="#", header=None).to_numpy()

experiment3 = pybamm.Experiment([
   ("Charge at 1C until 4.2V",
    "Hold at 4.2V until C/50",
    "Discharge at 1 C for 7 minutes",  
    "Run US06_A (A) for 3 hours",
    "Discharge at 0.2C until 2.7 V")
    ] * 20,
    drive_cycles={ "US06_A": drive_cycle_current,},
    termination="80% capacity",
    )

US06 drive cycle has been taken from the pybamm input folder. 
While the simulation ran, the cell is degrading extremely fast (lost 10% capacity in 16 cycles itself). Also, the voltage fluctuations are becoming bigger every cycle. I am not sure i understand why this is happening. May be my current values are too large for the cell. But how do I create a more appropriate profile. Some guidance would be helpful.

I have attached the images of the plots.

![image](https://user-images.githubusercontent.com/119900847/225686821-9031004a-86cb-47fc-a28f-7511259d81f9.png)

![image](https://user-images.githubusercontent.com/119900847/225686979-f906baf4-65cf-4ecf-8c45-0208e35251c5.png)

**Comment by RuiheLi** [comment]
Hi~

I think the OKane2022 parameter relates to a [paper ](https://pubs.rsc.org/en/content/articlehtml/2022/cp/d2cp00417h) which uses solvent-diffusion limited SEI. Therefore, it is likely that you get very quick degradation when switching to ec reaction limited SEI. According to the source code [here](https://github.com/pybamm-team/PyBaMM/blob/develop/pybamm/models/submodels/interface/sei/sei_growth.py#:~:text=k_exp%20%3D%20phase_param.k_sei%20*%20pybamm.exp,c_0%20/%20(1%20%2B%20L_over_D%20*%20k_exp)), there are two ageing paramters that control the degradation rate  of this ec reaction limited, k_sei (k_sei ) and D_ec (EC diffusivity [m2.s-1]). You can start from changing these values. Also if you have turned on lithium plating and LAM, there are more ageing parameters to tune, such as "Dead lithium decay constant [s-1]", 'Lithium plating kinetic rate constant [m.s-1]', "Negative electrode LAM constant proportional term [s-1]".
